src/lib/updater.ts

```
@@ -772,11 +808,210 @@ async function runOpenclawDoctorFix(): Promise<void> {
+    if (masked) {
+      try {
+        await removeGatewayMaintenanceMask();
+      } catch (unmaskErr) {
+        console.error(
+          "[Updater] Failed to remove the gateway maintenance mask:",
+          unmaskErr instanceof Error ? unmaskErr.message : unmaskErr,
+        );
+        // Preserve the operation's real error when both failed. If cleanup is
+        // the only failure, surface it: reporting success would leave the
+        // gateway runtime-masked until reboot.
+        if (!operationFailed) throw unmaskErr;
```

Biome (2.5.8)

[error] Unsafe usage of 'throw'.

(lint/correctness/noUnsafeFinally)